### ISSUE TITLE
Flutter 2.0: CanvasKit Path constructor invoked with wrong number of params

### ISSUE THREAD

**Issue by tiloc** [flutter/flutter#77177]
## Steps to Reproduce
Install fl_charts example app from GitHub: https://github.com/imaNNeoFighT/fl_chart.git
Change to branch: dev-nullsafety
Run on Chrome with "run -d chrome"

This is happening on both the Beta channel as well as the Stable channel.

**Expected results:**
The example app is running

**Actual results:**

<details>
  <summary>Logs</summary>

```
https://api.flutter.dev/flutter/dart-ui/ChannelBuffers-class.html
══╡ EXCEPTION CAUGHT BY RENDERING LIBRARY
╞═════════════════════════════════════════════════════════
The following NativeJavaScriptObject object was thrown during paint():
  BindingError: Tried to invoke ctor of Path with invalid number of parameters (1)
  - expected (0)
parameters instead!

The relevant error-causing widget was:
  CustomPaint-[GlobalKey#bd8a1]
  file:///Users/tilo/Projects/fl_chart/lib/src/chart/line_chart/line_chart.dart:143
  :16

When the exception was thrown, this was the stack:
https://unpkg.com/canvaskit-wasm@0.24.0/bin/canvaskit.js 154:76          <fn>
https://unpkg.com/canvaskit-wasm@0.24.0/bin/canvaskit.js 153:335         new
BindingError
https://unpkg.com/canvaskit-wasm@0.24.0/bin/canvaskit.js 205:496         <fn>
https://unpkg.com/canvaskit-wasm@0.24.0/bin/canvaskit.js 153:335         new Path
lib/_engine/engine/canvaskit/path.dart 16:15                             from
lib/ui/src/ui/path.dart 18:28                                            from
packages/fl_chart/src/chart/line_chart/line_chart_painter.dart 459:31
[_generateBelowBarPath]
packages/fl_chart/src/chart/line_chart/line_chart_painter.dart 200:28
[_drawBarLine]
packages/fl_chart/src/chart/line_chart/line_chart_painter.dart 106:7     paint
packages/flutter/src/rendering/custom_paint.dart 564:12
[_paintWithPainter]
packages/flutter/src/rendering/custom_paint.dart 605:7                   paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/shifted_box.dart 72:14                    paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/box.dart 2805:14
defaultPaint
packages/flutter/src/rendering/flex.dart 1079:7                          paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/box.dart 2805:14
defaultPaint
packages/flutter/src/rendering/stack.dart 629:5                          paintStack
packages/flutter/src/rendering/stack.dart 639:7                          paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/proxy_box.dart 2165:11                    paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/shifted_box.dart 72:14                    paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 140:10
_repaintCompositedChild
packages/flutter/src/rendering/object.dart 100:5
repaintCompositedChild
packages/flutter/src/rendering/object.dart 206:7
[_compositeChild]
packages/flutter/src/rendering/object.dart 187:7                         paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/sliver_multi_box_adaptor.dart 641:16      paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/sliver_padding.dart 274:14                paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/viewport.dart 650:16
[_paintContents]
packages/flutter/src/rendering/object.dart 396:12                        pushLayer
packages/flutter/src/rendering/object.dart 452:7
pushClipRect
packages/flutter/src/rendering/viewport.dart 637:31                      paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 140:10
_repaintCompositedChild
packages/flutter/src/rendering/object.dart 100:5
repaintCompositedChild
packages/flutter/src/rendering/object.dart 206:7
[_compositeChild]
packages/flutter/src/rendering/object.dart 187:7                         paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/widgets/basic.dart 7694:14                          paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 140:10
_repaintCompositedChild
packages/flutter/src/rendering/object.dart 100:5
repaintCompositedChild
packages/flutter/src/rendering/object.dart 206:7
[_compositeChild]
packages/flutter/src/rendering/object.dart 187:7                         paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/sliver_multi_box_adaptor.dart 641:16      paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/sliver_padding.dart 274:14                paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/viewport.dart 650:16
[_paintContents]
packages/flutter/src/rendering/object.dart 396:12                        pushLayer
packages/flutter/src/rendering/object.dart 452:7
pushClipRect
packages/flutter/src/rendering/viewport.dart 637:31                      paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 140:10
_repaintCompositedChild
packages/flutter/src/rendering/object.dart 100:5
repaintCompositedChild
packages/flutter/src/rendering/object.dart 206:7
[_compositeChild]
packages/flutter/src/rendering/object.dart 187:7                         paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/shifted_box.dart 72:14                    paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/box.dart 2805:14
defaultPaint
packages/flutter/src/rendering/custom_layout.dart 412:5                  paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/material/material.dart 551:11                       paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 396:12                        pushLayer
packages/flutter/src/rendering/proxy_box.dart 1925:14                    paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 140:10
_repaintCompositedChild
packages/flutter/src/rendering/object.dart 100:5
repaintCompositedChild
packages/flutter/src/rendering/object.dart 206:7
[_compositeChild]
packages/flutter/src/rendering/object.dart 187:7                         paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/box.dart 2805:14
defaultPaint
packages/flutter/src/rendering/stack.dart 629:5                          paintStack
packages/flutter/src/rendering/stack.dart 639:7                          paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/proxy_box.dart 2165:11                    paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/proxy_box.dart 2715:13                    paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/proxy_box.dart 2715:13                    paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 140:10
_repaintCompositedChild
packages/flutter/src/rendering/object.dart 100:5
repaintCompositedChild
packages/flutter/src/rendering/object.dart 206:7
[_compositeChild]
packages/flutter/src/rendering/object.dart 187:7                         paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/proxy_box.dart 3375:11                    paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/widgets/overlay.dart 780:14                         paintStack
packages/flutter/src/widgets/overlay.dart 792:7                          paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/custom_paint.dart 608:11                  paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/proxy_box.dart 142:14                     paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 189:12                        paintChild
packages/flutter/src/rendering/view.dart 200:14                          paint
packages/flutter/src/rendering/object.dart 2322:7
[_paintWithContext]
packages/flutter/src/rendering/object.dart 140:10
_repaintCompositedChild
packages/flutter/src/rendering/object.dart 100:5
repaintCompositedChild
packages/flutter/src/rendering/object.dart 978:29                        flushPaint
packages/flutter/src/rendering/binding.dart 455:19                       drawFrame
packages/flutter/src/widgets/binding.dart 874:13                         drawFrame
packages/flutter/src/rendering/binding.dart 319:5
[_handlePersistentFrameCallback]
packages/flutter/src/scheduler/binding.dart 1144:15
[_invokeFrameCallback]
packages/flutter/src/scheduler/binding.dart 1082:9
handleDrawFrame
packages/flutter/src/scheduler/binding.dart 865:7                        <fn>
dart-sdk/lib/_internal/js_dev_runtime/private/isolate_helper.dart 48:19
internalCallback

The following RenderObject was being processed when the exception was fired:
RenderCustomPaint#05a25:
  creator: CustomPaint-[GlobalKey#bd8a1] ← Listener ← _GestureSemantics ←
  RawGestureDetector ←
    GestureDetector ← _RawMouseRegion ← MouseRegion ← LineChart ← Padding ←
    Expanded ← Column ← Stack
    ← ⋯
  parentData: <none> (can use size)
  constraints: BoxConstraints(w=567.0, h=333.9)
  size: Size(567.0, 333.9)
This RenderObject has no descendants.
═══════════════════════════════════════════════════════════════════════════════════
═════════════════
Another exception was thrown: BindingError: Tried to invoke ctor of Path with
invalid number of parameters (1) - expected (0) parameters instead!
Another exception was thrown: BindingError: Tried to invoke ctor of Path with
invalid number of parameters (1) - expected (0) parameters instead!
Another exception was thrown: BindingError: Tried to invoke ctor of Path with
invalid number of parameters (1) - expected (0) parameters instead!
Another exception was thrown: BindingError: Tried to invoke ctor of Path with
invalid number of parameters (1) - expected (0) parameters instead!
Another exception was thrown: BindingError: Tried to invoke ctor of Path with
invalid number of parameters (1) - expected (0) parameters instead!
```

```
flutter analyze
Analyzing example...

   info • `Future` results in `async` function bodies must be `await`ed or marked
          `unawaited` using `package:pedantic` •
          lib/bar_chart/samples/bar_chart_sample1.dart:307:7 • unawaited_futures
   info • Omit type annotations for local variables •
          lib/bar_chart/samples/bar_chart_sample2.dart:115:35 •
          omit_local_variable_types
   info • Omit type annotations for local variables •
          lib/bar_chart/samples/bar_chart_sample2.dart:116:40 •
          omit_local_variable_types
   info • Omit type annotations for local variables •
          lib/bar_chart/samples/bar_chart_sample2.dart:214:5 •
          omit_local_variable_types
   info • Omit type annotations for local variables •
          lib/bar_chart/samples/bar_chart_sample2.dart:215:5 •
          omit_local_variable_types
   info • Omit type annotations for local variables •
          lib/line_chart/samples/line_chart_sample1.dart:181:5 •
          omit_local_variable_types
   info • Omit type annotations for local variables •
          lib/line_chart/samples/line_chart_sample1.dart:204:5 •
          omit_local_variable_types
   info • Omit type annotations for local variables •
          lib/line_chart/samples/line_chart_sample1.dart:226:5 •
          omit_local_variable_types
   info • 'FlatButton' is deprecated and shouldn't be used. Use TextButton instead.
          See the migration guide in
          flutter.dev/go/material-button-migration-guide). This feature was
          deprecated after v1.25.0-8.1.pre. •
          lib/line_chart/samples/line_chart_sample2.dart:40:18 •
          deprecated_member_use
   info • Omit type annotations for local variables •
          lib/line_chart/samples/line_chart_sample3.dart:56:23 •
          omit_local_variable_types
   info • Omit type annotations for local variables •
          lib/line_chart/samples/line_chart_sample5.dart:49:5 •
          omit_local_variable_types
   info • Omit type annotations for local variables •
          lib/line_chart/samples/line_chart_sample6.dart:29:10 •
          omit_local_variable_types
   info • Omit type annotations for local variables •
          lib/line_chart/samples/line_chart_sample8.dart:24:5 •
          omit_local_variable_types
   info • Omit type annotations for local variables •
          lib/line_chart/samples/line_chart_sample8.dart:25:5 •
          omit_local_variable_types
   info • Omit type annotations for local variables •
          lib/line_chart/samples/line_chart_sample8.dart:26:5 •
          omit_local_variable_types
   info • Omit type annotations for local variables •
          lib/line_chart/samples/line_chart_sample8.dart:31:5 •
          omit_local_variable_types
   info • Omit type annotations for local variables •
          lib/line_chart/samples/line_chart_sample8.dart:34:5 •
          omit_local_variable_types
   info • Omit type annotations for local variables •
          lib/line_chart/samples/line_chart_sample8.dart:36:5 •
          omit_local_variable_types
   info • This method overrides a method annotated as '@mustCallSuper' in 'State',
          but doesn't invoke the overridden method • lib/main.dart:57:8 •
          must_call_super
   info • Omit type annotations for local variables •
          lib/pie_chart/samples/pie_chart_sample1.dart:98:9 •
          omit_local_variable_types
   info • Omit type annotations for local variables •
          lib/pie_chart/samples/pie_chart_sample2.dart:100:7 •
          omit_local_variable_types
   info • Omit type annotations for local variables •
          lib/pie_chart/samples/pie_chart_sample2.dart:101:7 •
          omit_local_variable_types
   info • Omit type annotations for local variables •
          lib/pie_chart/samples/pie_chart_sample3.dart:50:7 •
          omit_local_variable_types
   info • Omit type annotations for local variables •
          lib/pie_chart/samples/pie_chart_sample3.dart:51:7 •
          omit_local_variable_types
   info • Omit type annotations for local variables •
          lib/pie_chart/samples/pie_chart_sample3.dart:52:7 •
          omit_local_variable_types
warning • Publishable packages can't have path dependencies • pubspec.yaml:24:5 •
       invalid_dependency

26 issues found. (ran in 4.8s)
```

```
flutter doctor -v
[✓] Flutter (Channel beta, 2.0.0, on macOS 11.2.1 20D74 darwin-arm, locale en-DE)
    • Flutter version 2.0.0 at /Users/tilo/development/flutter
    • Framework revision 60bd88df91 (5 hours ago), 2021-03-03 09:13:17 -0800
    • Engine revision 40441def69
    • Dart version 2.12.0

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/tilo/Library/Android/sdk
    • Platform android-30, build-tools 30.0.3
    • ANDROID_SDK_ROOT = /Users/tilo/Library/Android/sdk
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6915495)

[✓] IntelliJ IDEA Community Edition (version 2020.3.2)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 54.0.3
    • Dart plugin version 203.7759

[✓] Connected device (3 available)
    • Pixel 5 (mobile) • 08081FDD4002T6 • android-arm64  • Android 11 (API 30)
    • macOS (desktop)  • macos          • darwin-arm64   • macOS 11.2.1 20D74
      darwin-arm
    • Chrome (web)     • chrome         • web-javascript • Google Chrome
      88.0.4324.192

• No issues found!
```

</details>

**Comment by tiloc** [comment]
FWIW: The dev channel does not have this issue. Works perfect with the version as listed below:

```
flutter doctor -v
[✓] Flutter (Channel dev, 2.1.0-10.0.pre, on macOS 11.2.1 20D74 darwin-arm,
    locale en-DE)
    • Flutter version 2.1.0-10.0.pre at /Users/tilo/development/flutter
    • Framework revision cc9b78fc5c (6 days ago), 2021-02-25 13:26:03 -0800
    • Engine revision a252ec09b7
    • Dart version 2.13.0 (build 2.13.0-77.0.dev)

[✓] Android toolchain - develop for Android devices (Android SDK version
    30.0.3)
    • Android SDK at /Users/tilo/Library/Android/sdk
    • Platform android-30, build-tools 30.0.3
    • ANDROID_SDK_ROOT = /Users/tilo/Library/Android/sdk
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6915495)

[✓] IntelliJ IDEA Community Edition (version 2020.3.2)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 54.0.3
    • Dart plugin version 203.7759

[✓] Connected device (3 available)
    • Pixel 5 (mobile) • 08081FDD4002T6 • android-arm64  • Android 11 (API
      30)
    • macOS (desktop)  • macos          • darwin-arm64   • macOS 11.2.1 20D74
      darwin-arm
    • Chrome (web)     • chrome         • web-javascript • Google Chrome
      88.0.4324.192

• No issues found!
```

**Comment by darshankawar** [comment]
@tiloc 
This is similar to https://github.com/flutter/flutter/issues/75473 which has been fixed and merged in master. Please switch to latest master and try again to confirm.
The fix may not yet have reached beta and stable and hence you may be seeing the issue in those channels. https://github.com/imaNNeoFighT/fl_chart/issues/561#issuecomment-790088310

**Comment by tiloc** [comment]
Not sure how to "switch to latest master". as mentioned by me earlier, this issue does not exist on dev channel. What is the typical cadence for fixes to move from dev to stable?

**Comment by yjbanov** [comment]
We have roughly 4 stable releases per year. beta comes out once a month or so. Dev is continuous, but a little behind master as it must pass additional customer tests.

Sorry, I thought that this fix would make it into this stable release on its own and so I didn't request a cherry pick. I'll see if I can request a hot-fix, but can't promise anything. Hot-fix bar is quite high.

**Comment by darshankawar** [comment]
@tiloc 
`flutter channel master`
`flutter upgrade`

**Comment by yjbanov** [comment]
I requested a cherry pick for 2.0.1, which should be released much sooner than 2.1.0 (not sure exactly when, but I think we're counting days, not weeks).

**Comment by yjbanov** [comment]
I'm closing the issue as the fix is making its way to the release channels already.
